e2e/e2e_ext_test.go

```diff
@@ -2272,6 +2272,166 @@ func TestPopPayouts(t *testing.T) {
 	}
 }
 
+func TestPopPayoutsMultiplePages(t *testing.T) {
```

Comment:
nit: I think some of the err assignments in this test could be nested into the if statement, so long as they don't exceed 80 characters in length.